scanners/nikto/examples/demo-bodgeit/nikto-results.json

```diff
@@ -1,57 +1,601 @@
 {
```

Comment:
I think we can get rid of this file. In the examples folder we don't need any raw results, only in the tests. Please also delete the juice-shop one.